app/src/main/java/org/ieeeguc/ieeeguc/fragments/user/UserShow.java

```diff
@@ -1,23 +1,94 @@
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_user_show, container, false);
+        final View view = inflater.inflate(R.layout.fragment_user_show, container, false);
+        MainActivity.loggedInUser.getUser(MainActivity.token,MainActivity.loggedInUser.getId(), new HTTPResponse() {
+            @Override
+            public void onSuccess(int statusCode, JSONObject body) {
+                try {
+                    JSONObject j = body.getJSONObject("user");
+
+                    int id = j.getInt("id");
+                   User.Type type = User.Type.valueOf(j.getString("type").toUpperCase());
+                   String first_name  = j.getString("first_name");
+                   String last_name = j.getString("last_name");
+                   String email = j.getString("email");
+                   User.Gender gender = User.Gender.valueOf(j.getString("gender").toUpperCase());
+                   String phone_number = j.getString("phone_number");
+                    String birthdate = j.getString("birthdate");
+                   String IEEE_membership_ID = j.getString("IEEE_membership_ID");
+                    JSONObject settings = j.getJSONObject("settings");
+                    SimpleDateFormat formate = new SimpleDateFormat("yyyy-MM-dd");
+                    Date date = formate.parse(birthdate);
+
+                    u = new User(id,type,first_name, last_name,gender,email,date,
+                            IEEE_membership_ID, 0, null, phone_number,settings);
+
+
+                    ((TextView) view.findViewById(R.id.name)).setText(u.getFirstName()+" "+u.getLastName());
+                    ((TextView) view.findViewById(R.id.email)).setText(u.getEmail());
+                    ((TextView) view.findViewById(R.id.id)).setText(u.getId()+"");
+                    ((TextView) view.findViewById(R.id.ieeemsh_id)).setText(u.getIeeeMembershipID());
+                    ((TextView) view.findViewById(R.id.phone)).setText(u.getPhoneNumber());
+                    ((TextView) view.findViewById(R.id.type)).setText(u.getType().toString());
+                    Date dat = new Date();
+                    String bdate = formate.format(dat);
+                    ((TextView) view.findViewById(R.id.bdate)).setText(bdate);
+                    ((TextView) view.findViewById(R.id.gender)).setText(u.getGender().toString());
+
+                    } catch (Exception e) {
+                    MainActivity.createSnackBar(getString(R.string.error_server_down));
+                }
+
+            }
+
+            @Override
+            public void onFailure(int statusCode, JSONObject body) {
+
+                if(statusCode == 401) {
+                    MainActivity.logout();
+                }
+                else if(statusCode == 400 || statusCode == 403)

```

Comment:
403 should be handeled using a not authorized message i would suggest the following `Access denied, you are not allowed to view the requested resource. `